Content.Server/Commands/Observer/Ghost.cs

```diff
@@ -27,71 +27,11 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
         {
             if (player == null)
             {
-                shell.SendText(player, "Nah");
+                shell?.SendText((IPlayerSession?) null, "Nah");
```

Comment:
why not use `player` here?

Reply:
I think (not 100% sure) that this was changed back during porting of the previous changes

Reply:
Done.